node/directives/checkUserAccess.ts

```diff
@@ -157,7 +16,155 @@ export class CheckUserAccess extends SchemaDirectiveVisitor {
       context: Context,
       info: any
     ) => {
-      await checkUserOrAdminTokenAccess(context, field.astNode?.name?.value)
```

Comment:
Removed this extra function call as there was no real gain with it and to keep same style as `checkAdminAccess` directive